schema.sql

```diff
@@ -7,5 +7,6 @@ date_of_birth date NOT NULL,
 escape_attempts int,
 neutered boolean, 
 weight_kg decimal,
+species varchar,
```

Comment:
- [ ] Instead of adding the column manually in the table, please be mindful that the purpose of this requirement is to use [`ALTER TABLE`](https://www.postgresql.org/docs/current/sql-altertable.html) SQL command. Kindly add the column this way.